src/NBallot.php

```diff
@@ -3,19 +3,19 @@
 
 /**
```

Comment:
This class inherits a method `count()` from its parent class that counts the number lists of candidates.  That method name is really close to this class' `getCount()` method, but `getCount()` returns a totally different number --a BFF multiplier i.e. (42 * A > B > C). To make the distinction clearer I renamed this class' `getCount()` method to `getMultiplier()` and updates associated variable names.